packages/Php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Rector\Php70\Tests\Rector\FuncCall\NonVariableToVariableOnFunctionCallRector\Fixture;
+
+class Child
+{
+    public function bar(&$bar) {}
+}
+
+class Dummy
+{
+    public static $bar;
+    public $baz;
+
+    public function baz(&$bar) {}
+
+    public function child(): Child
+    {
+        return new Child();
+    }
+}
+
+function byRef($bar, &$baz) {}
+
+function allByRef(&$bar, &$baz) {}
+
+function aFunction()
+{
+    reset($var);
+    reset([1][0]);
+    reset(Dummy::$bar);
+    $dummy = new Dummy();
+    reset($dummy->baz);
+
+    reset($var = []);
+
+    reset(bar());
+    byRef(bar(), baz());
+    allByRef(bar(), baz());
+
+    reset(new Dummy());
+
+    $anonymousFunction = function (&$bar) {};
+    $staticAnonymousFunction = static function (&$bar) {};
+    $anonymousFunction(bar());
+    $staticAnonymousFunction(bar());
+
+    $dummy = new Dummy();
+    $dummy->baz(baz());
+    $dummy->child()->bar(bar());
+
+    $anonymousClass = new class {
+        public function bar(&$baz) {}
+    };
+    $anonymousClass->bar(baz());
+}
+
```

Comment:
It's best practise here to use more smaller test fixtures, instead of only 1 with all the use cases.

If one case breaks here in the future, it's very hard to find the bug. Having 10 files would make that easier